collectors/o365/o365_collector.js

```diff
@@ -99,12 +98,16 @@ class O365Collector extends PawsCollector {
             return callback(null, [], state, state.poll_interval_sec);
         }
 
+        if ((process.env.paws_collection_interval && process.env.paws_collection_interval > 0 ) && state ) {
+            state.until = moment(state.since).add(process.env.paws_collection_interval, 'seconds').toISOString();
+        }
         console.info(`O365000001 Collecting data from ${state.since} till ${state.until} for stream ${state.stream}`);
 
         if(moment().diff(state.since, 'days', true) > 7){
             const newStart = moment().subtract(PARTIAL_WEEK_HOURS, 'hours');
             state.since = newStart.toISOString();
-            state.until = newStart.add(1, 'hours').toISOString();
+            state.until = process.env.paws_collection_interval && process.env.paws_collection_interval > 0 ? newStart.add(process.env.paws_collection_interval, 'seconds').toISOString()
+                : newStart.add(1, 'hours').toISOString();
```

Comment:
we don't need this change. Simply move [this if](https://github.com/alertlogic/paws-collector/pull/200/files#diff-458b6045c8dfaadace71e6dbd57f1b5625d5c3f985af520c6b352b5111cff86cR101-R103) to [here](https://github.com/alertlogic/paws-collector/pull/200/files#diff-458b6045c8dfaadace71e6dbd57f1b5625d5c3f985af520c6b352b5111cff86cR120) - after current if statement